### ISSUE TITLE
DeepLabV3+ Xception SemanticPredictions output layer corrupt

### ISSUE THREAD

**Issue by thhart** [tensorflow/tensorflow#44375]
I can not do inference on a frozen custom trained Xception model. Resnet/Mobilenet is working ok. What might be the difference then? Is there anything special to know about the Xception model output?
During training loss and images look very good. However when using the frozen model the output is always empty.

**System information**
Platform: Linux Ubuntu 20.04 / Nvidia GPU 8 GB
Python: 3.7
Tensorflow: v1.15.2-30-g4386a66
Model: xception_41

**Describe the current behavior**
Output tensor for SemanticPredictions is always the same value.

**Describe the expected behavior**
Output tensor for SemanticPredictions should contain the classes prediction value.

**Other info / logs** Include any logs or source code that would be helpful to

The model was trained:
```
python deeplab/train.py --logtostderr --training_number_of_steps=96400 \
  --train_split="trainval"  \
  --train_crop_size="384,384" --train_batch_size=4 --dataset="vehicle" --save_interval_secs=300 \
  --save_summaries_secs=300 --save_summaries_images=True --log_steps=100 --train_logdir=training_xception_41 \
  --dataset_dir=${PATH_TO_DATASET} \
  --scale_factor_step_size=0.05 \
  --min_scale_factor=0.60 \
  --max_scale_factor=0.75 \
  --scale_factor_step_size=0.05 \
  --output_stride=8 --base_learning_rate=0.125 \
  --weight_decay=0.00125 \
  --model_variant="xception_41"
```

The model was frozen:
```
python deeplab/export_model.py \
--model_variant="xception_41" \
--num_classes=${CLASSES} \
--dataset="vehicle" \
--checkpoint_path=${toFreeze} \
--crop_size=384 \
--crop_size=384 \
--export_path=graph_xception_41.pb \
--output_stride=8
```

I can see a difference in output graphs (left Resnet / right Xception):
[![https://imgur.com/pPswXVo.png](https://imgur.com/pPswXVo.png)](https://imgur.com/pPswXVo.png)

**Comment by amahendrakar** [comment]
@thhart,
Issues related to tensorflow/models are tracked in the models repo. Could you please submit a new issue from [this link](https://github.com/tensorflow/models/issues/new/choose), so that the issue can be tracked there. Thanks!

**Comment by thhart** [comment]
The same problem applies to nas_pnasnet by the way.
However I can see a difference SemanticProbabilities layer here also empty.
For Xception the SemanticProbabilities layer was not empty.

**Comment by thhart** [comment]
Moved to: https://github.com/tensorflow/models/issues/9426
